### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by smezger**
* fix Readme.md, reference correct repo
* fix reference of css files to give custom.css highest prio
* fix typos in localization.js

### CODE REVIEW

File: site/js/localization.js [hunk 1]
```diff
@@ -257,8 +257,8 @@ function prettyPrintDateString(date) {
     var d = new Date(date)
     let localeDate = d.toLocaleString(getLocale(), {
         weekday: "long",
-        day: "numeric",
-        year: "numeric",
+        day: "Numberic",
```
Comment (BorisBrock) [inline site/js/localization.js:260]: Is "Numberic" a valid option? According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString, "numeric" seems to be correct. Am I missing something here?

File: site/js/localization.js [hunk 2]
```diff
@@ -267,7 +267,7 @@ function prettyPrintDateString(date) {
 function prettyPrintDateStringWithoutDay(date) {
     var d = new Date(date)
     let localeDate = d.toLocaleString(getLocale(), {
-        year: "numeric",
+        year: "Numberic",
```
Comment (BorisBrock) [inline site/js/localization.js:270]: See previous comment.

**Review by BorisBrock — CHANGES_REQUESTED** [review]
Hi @smezger, thanks your your PR. Could you clarify on the "Numberic" option?

File: site/js/localization.js (hunk 1 above)
Comment (smezger) [reply site/js/localization.js:260]: Just a stupid search and replace error ..😒

File: site/js/localization.js (hunk 2 above)
Comment (smezger) [reply site/js/localization.js:270]: see above

**Pull request merged**